omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -31,57 +36,89 @@ Install ZeroC IcePy 3.6. IcePy is managed by PyPI and will be installed as a par
         libdb-utils \
         openssl-devel bzip2-devel expat-devel
 
-Install other dependencies (run as root)::
+Install other dependencies. The number of dependencies to install depends on the way you plan to install OMERO.web. If you wish to install it in a virtual environment created with ``--system-site-packages`` *on* (**option 1**), you will need to install ``python-pillow`` and ``numpy``. If you wish to install it in a virtual environment with ``--system-site-packages`` *off*, few more dependencies will be required (**option 2**)::
     
+    # common dependencies to both options
```

Comment:
swap, "dependencies common to ..."

Reply:
Fixed in https://github.com/ome/omeroweb-install/pull/22